### ISSUE TITLE
Support Sphere in preset shapes supplied to SetSample

### ISSUE THREAD

**Issue by RichardWaiteSTFC** [mantidproject/mantid#32892]
**Describe the solution you'd like**
At present there are some shapes (e.g. hollow cylinder) that are preset and can be created in `SetSample `like so
```
ws = CreateSampleWorkspace()
SetSample(ws, Geometry={'Shape': 'HollowCylinder', 'Height': 4.0, 'InnerRadius': 0.8, 'OuterRadius': 1.0, 'Center': [0.,0.,0.]},
    Material={'ChemicalFormula': 'V0.95-Nb0.05', 'SampleNumberDensity': 0.0119})
```
However to make a sphere we need to provide an xml string
```
shape= '''<sphere id="V-sphere">
<centre x="0.0"  y="0.0" z="0.0" />
<radius val="0.0025"/>
</sphere>'''
CreateSampleShape('SXD33338_sm_deadCor', shape)
SetSampleMaterial('SXD33338_sm_deadCor', SampleNumberDensity=0.0119, 
    ScatteringXSection = 5.197,AttenuationXSection=4.739,ChemicalFormula='V0.95 Nb0.05')
```
The solution I would like would be to support spherical shapes as so
```
SetSample('SXD33338_sm_deadCor', Geometry={'Shape': 'Sphere', 'Radius': 0.0025},
          Material={'ChemicalFormula': 'V0.95-Nb0.05', 'SampleNumberDensity': 0.0119}) 
```

**Describe alternatives you've considered**
Alternatively (additionally?) would it be possible to provide a better error to inform the user the shape provided is invalid.
Running this code 
```
SetSample('SXD33338_sm_deadCor', Geometry={'Shape': 'Sphere', 'Radius': 0.0025},
          Material={'ChemicalFormula': 'V0.95-Nb0.05', 'SampleNumberDensity': 0.0119}) 
```
Throws
```
Invalid value for Geometry: For Sphere shape Height is required
RuntimeError: SetSample-v1: Some invalid Properties found: [ Geometry ]
```
It would be better to say 'sphere' is not supported.
Note if you do call SetSampel with a Height for sphere (as the message suggests) 
```
SetSample('SXD33338_sm_deadCor', Geometry={'Shape': 'Sphere', 'Height': 0.0025},
          Material={'ChemicalFormula': 'V0.95-Nb0.05', 'SampleNumberDensity': 0.0119})  
```   
you get a more appropriate error
```
Error in execution of algorithm SetSample:
Unknown 'Shape' argument 'Sphere' provided in 'Geometry' property. Allowed values are CSG, FlatPlate, Cylinder, HollowCylinder, FlatPlateHolder, HollowCylinderHolder
ValueError: Unknown 'Shape' argument 'Sphere' provided in 'Geometry' property. 
```

**Additional context**

**Comment by DannyHindson** [comment]
check radius > 0

**Comment by DannyHindson** [comment]
David seemed to get quite close to adding this a few years ago in this PR but don't think it was ever merged: #20237.

**Comment by gvardany** [comment]
Sphere should be supported with Radius of course and not height. I think the misleading error just comes from the fact, that for all the existing shapes (flat, cylinder, annulus) height is the common mandatory parameter.
